base/applications/mscutils/servman/propsheet_logon.c

```diff
@@ -137,7 +137,7 @@ OnQueryInitialFocus(
 {
     HWND hwnd = GetDlgItem(hwndDlg, pLogonData->bLocalSystem ? IDC_LOGON_SYSTEMACCOUNT : IDC_LOGON_THISACCOUNT);
 
-    SetWindowLong(hwndDlg, DWL_MSGRESULT, (LPARAM)hwnd);
+    SetWindowLongPtr(hwndDlg, DWLP_MSGRESULT, (LPARAM)hwnd);
```

Comment:
This is technically not LPARAM but LONG_PTR.

Reply:
And technichally LPARAM is LONG_PTR?

Reply:
What hermes means, is that here you use `(LPARAM)`, but in `dll/cpl/intl/advanced.c` you use `(LONG_PTR)`